pkg/drivers/driver.go

```diff
@@ -79,3 +79,13 @@ func convertEnvToMap(env []string) map[string]string {
 	}
 	return envMap
 }
+
+func convertLabelsToMap(labels []string) map[string]string {
```

Comment:
this logic is identical to `convertEnvToMap`, can you just rename that function to `convertSliceToMap` or something and reuse that here?

Reply:
can do